src/local/local_config.js.sample

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+// Local overrides to config, modify this instead of global config.
+
+//config.profiler.enabled = false;
+//config.stats.enabled = true;
+//config.stats.summary = true;
+//config.layout.plainCost = 4;
+//config.layout.swampCost = 7;
+//config.info.signController = false;
+
+// TODO: need to add config.layout.roadCost
+//config.layout.roadCost = 3;
```

Comment:
nitpicking: newline

Maybe the files in `src/local` are not jshinted, beautified, jscsed?